src/components/TermsOfUsePage.vue

```diff
@@ -1,5 +1,15 @@
 <template>
 <div class="grid-container body-search-content">
+  <PageHeading title="Introduction"/>
+
+  <div class="grid-content large-3 columns about-text">
+    <p class="landing-text">You have started a health questionnaire that asks about various symptoms associated with rare diseases and genetic conditions. This questionnaire is for patients with rare conditions to describe their physical features and disease characteristics, which may help clinicians and researchers find a diagnosis. This information may help further the understanding of what possible effects specific genetic changes may have on health and development. This tool does not provide medical advice. It is intended for informational and educational purposes only.</p>
+    <p class="landing-text">The survey should take 10-15 minutes to complete. <b>Answers are not saved until you click the “submit” button at the end of the survey.</b></p>
+    <p class="landing-text">If you would like to ask a question before finishing this survey, please email <a href="mailto:MyTraitFinderStudy@childrens.harvard.edu.">MyTraitFinderStudy@childrens.harvard.edu</a>.</p>
+    <p class="landing-text">You have agreed to participate and will now move on to the health survey. Please remember that as you answer the questions, you should answer them from the perspective of the person with the rare disease or genetic condition who consented to participate. If you are answering on behalf of your child, for example, when we say “YOU” we are meaning “your child”.</p>
+  </div>
+  <br/>
+
   <PageHeading title="Terms of Use"/>
```

Comment:
Using the `PageHeading` component for the terms of use looks kind of odd now. I might switch this to a simple h2 tag.